### ISSUE TITLE
Add extra output in the MFILE for ease of extraction to a spreadsheet in develop-stable

### ISSUE THREAD

**Issue by jonmaddock** [ukaea/PROCESS#1377]
In GitLab by @cjwgriesel on Aug 11, 2021, 17:01

Add extra output to the MFILE. Some output in the MFILE is calculated and needs a variable assigned to the result of the calculation to make it easy to extract the information from the MFILE. i.e
tot_power_plasma=falpha*palpmw+pchargemw+pohmmw+pinjmw       variable to create and output in the MFILE
normalised_toroidal_beta=normalised_total_beta*(btot/bt)**2  variable to create and output in the MFILE
sum = pnetelmw+pinjwp+htpmw+vachtmw+trithtmw+crypmw+tfacpd+fachtmw+pfwpmw
tot_plant_power=sum  variable to create and output in the MFILE
There maybe others to add?
In addition for the type of plasma heating selected ie NBI, ECRH/ECCD or EBW those variables need to be output in the MFILE but only the one that is selected should be shown in the MFILE:- 
etanbi
xi_ebw
gamma_ecrh

pes plasma heating

**Comment by jonmaddock** [comment]
Do we really want this?

**Comment by chris-ashe** [comment]
> Do we really want this?

I dont think we do. I don't see how this would be useful to anyone at the moment
